mission_planner1/UAVState[] other_UAVs_state